README.md

```diff
@@ -6,35 +6,92 @@ The version of the VMware Cloud Director API and Installation that are compatibl
 
 | CPI Version | VMware Cloud Director API | VMware Cloud Director Installation | Notes |
 | :---------: | :-----------------------: | :--------------------------------: | :---: |
-| 0.1.0.latest | 35.0+ | 10.2+ | Needs NSX Advanced Load Balancer (Avi) version 20.1.1+ |
+| 0.1.0-beta | 35.0+ | 10.3.0+ | Needs NSX-T 3.1.1 with NSX Advanced Load Balancer (Avi) version 20.1.3+ |
+| 0.1.0 | 36.1+ | 10.3.1 | Needs NSX-T 3.1.1 with NSX Advanced Load Balancer (Avi) version 20.1.3+ |
 
 This extension is intended to be installed into a Kubernetes cluster installed with [VMware Cloud Director](https://www.vmware.com/products/cloud-director.html) as a Cloud Provider, by a user that has the same rights as a [vApp Author](https://docs.vmware.com/en/VMware-Cloud-Director/9.7/com.vmware.vcloud.admin.doc/GUID-BC504F6B-3D38-4F25-AACF-ED584063754F.html).
 
-**cloud-provider-for-cloud-director** is distributed as a container image hosted at [Distribution Harbor](https://projects.registry.vmware.com) as `projects.registry.vmware.com/vmware-cloud-director/cloud-provider-for-cloud-director:<version>`.
+**cloud-provider-for-cloud-director** is distributed as a container image hosted at [Distribution Harbor](https://projects.registry.vmware.com) as `projects.registry.vmware.com/vmware-cloud-director/cloud-provider-for-cloud-director:<CPI Version>`.
 
 This cloud-provider is in a preliminary `alpha` state and is not yet ready to be used in production.
 
 ## VMware Cloud Director Configuration
 
+In this section, we assume that the Kubernetes cluster is created using the [Container Service Extension](https://github.com/vmware/container-service-extension). However that is not a mandatory requirement. We will later describe the process of enabling a user-created Kubernetes Cluster with CPI.
+
+NSX-T with NSX Advanced Load Balancer is a prerequisite to use LoadBalancers with CPI of VCD. 
+
+The Kubernetes cluster on which the CPI needs to be installed must be accessible to the user whose credentials are used by CPI. Let us call the latter user as the `CPI user`.
```

Comment:
question: is there only one user spoken about here? if so, I don't think the term "latter" is needed